github/repos.go

```diff
@@ -61,18 +61,18 @@ type Repository struct {
 	License *License `json:"license,omitempty"`
 
 	// Additional mutable fields when creating and editing a repository
-	Private           *bool   `json:"private"`
-	HasIssues         *bool   `json:"has_issues"`
-	HasWiki           *bool   `json:"has_wiki"`
-	HasPages          *bool   `json:"has_pages"`
+	Private           *bool   `json:"private,omitempty"`
+	HasIssues         *bool   `json:"has_issues,omitempty"`
+	HasWiki           *bool   `json:"has_wiki,omitempty"`
+	HasPages          *bool   `json:"has_pages,omitempty"`
+	HasDownloads      *bool   `json:"has_downloads,omitempty"`
 	HasProjects       *bool   `json:"has_projects,omitempty"`
-	HasDownloads      *bool   `json:"has_downloads"`
```

Comment:
What's the reason you re-arranged the order of the fields?

It's not alphabetical, and I can't quite see the pattern. If there's a good reason, that's fine. If not, it'd be better to keep their order unchanged and focus this PR on just adding `,omitempty`s.

Reply:
No reason. That's wrong, indeed. I mistakenly changed the order of fields while resolving the merge conflict with that new field (`HasProjects`).